bazel/envoy_build_system.bzl

```diff
@@ -38,7 +38,11 @@ def envoy_linkopts():
         # TODO(zuercher): build id could be supported via "-sectcreate __TEXT __build_id <file>"
         # The file could should contain the current git SHA (or enough placeholder data to allow
         # it to be rewritten by tools/git_sha_rewriter.py).
-        "@bazel_tools//tools/osx:darwin": [],
+        "@bazel_tools//tools/osx:darwin": [
+            # TODO(zuercher): should be able to remove this after the next gperftools release after
```

Comment:
Please add a link to the gperftools bug report (here and in `envoy_test_linkopts`).